extras/deploy/container/run.sh

```diff
@@ -1 +1 @@
-./Godot_v3.2.1-stable_linux_server.64 --main-pack data.pck
+./Godot_v3.2.1-stable_linux_server.64 --public --main-pack data.pck --name "Fugitive3D Official #1"
```

Comment:
Maybe the name should be an argument, so we can change it easily: `"Fugitive3D Official # 2"`